### DISCUSSION TITLE
Suggestion : WeakAuras like in World of Warcraft

### DISCUSSION THREAD

**Discussion by JustZerooo** [goatcorp/Dalamud#1082] [Ideas]
Weakauras is a powerfull multifunctional addon in World of Warcraft that allows you more then just display an Icon or move a healthbar. It is an Addon that is powerfull. it allows you to display some informations etc. Interface Grouping, Dependencies etc.

1 Example: Player cast Spell XYZ and Spell has Cooldown, you can make an Icon of it and make this animated and place inside a cooldown timer when spell is ready etc.

What is WeakAuras? -> https://www.curseforge.com/wow/addons/weakauras-2
Can i get a demo? -> https://www.youtube.com/watch?v=Olj2cgb8mlU

My own words cannot explain it like the youtube video. So please check it out. It will be great if we could do this project as community. Free 4 all if time allows to work on it.

**Comment by tommadness** [comment]
This is not the place for plugin suggestions. Use this repo: https://github.com/goatcorp/suggestions

**Comment by JustZerooo** [comment]
It is under require discussion : https://github.com/goatcorp/suggestions/issues/30
